src/surfing_penguin/routes.py

```diff
@@ -0,0 +1,44 @@
+from surfing_penguin import surfing_penguin
+from flask import render_template, flash, redirect
+from surfing_penguin.forms import LoginForm
+
+@surfing_penguin.route('/')
+@surfing_penguin.route('/index')
+def index():
+    return render_template('index.html')
+
+@surfing_penguin.route('/login', methods=['GET', 'POST'])
+def login():
+    form = LoginForm()
+    if form.validate_on_submit():
+        flash('Login requested for user {}, remember_me={}'.format(
+            form.username.data, form.remember_me.data))
+        return redirect(url_for('index'))
+    return render_template('login.html', title='Sign In', form=form)
+
+@surfing_penguin.route('/select_qstnr')
+def select_qstnr():
```

Comment:
Docstrings should be added.
Please see http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html